src/App.tsx

```diff
@@ -40,6 +49,18 @@ export default App
 
 const Nav = tw.nav`bg-gradient-to-r from-blue-300 to-purple-300 py-16`
 
-const NavLink = tw.span`inline my-0 mx-8 no-underline text-3xl font-semibold`
+const NavLink = styled.span`
+  ${tw`inline my-0 mx-8 no-underline text-3xl font-semibold`}
+
+  & svg {
+    ${tw`inline-block align-top`}
+  }
+`
+
+const BottomAlign = styled.span`
+  & svg {
```

Comment:
instead of using a selector here, I'd definitely just make ImSearch a styled component